lib/plugins/acts_as_tree/test/acts_as_tree_test.rb

```diff
@@ -181,17 +181,17 @@ def test_eager_association_loading
   end
 
   def test_eager_association_loading_with_recursive_cascading_three_levels_has_many
-    root_node = RecursivelyCascadedTreeMixin.find(:first, include: { children: { children: :children } }, order: 'mixins.id')
+    root_node = RecursivelyCascadedTreeMixin.includes({ children: { children: :children } }).order('mixins.id').first
     assert_equal @rc4, assert_no_queries { root_node.children.first.children.first.children.first }
   end
 
   def test_eager_association_loading_with_recursive_cascading_three_levels_has_one
-    root_node = RecursivelyCascadedTreeMixin.find(:first, include: { first_child: { first_child: :first_child } }, order: 'mixins.id')
+    root_node = RecursivelyCascadedTreeMixin.includes({ first_child: { first_child: :first_child } }).order('mixins.id').first
```

Comment:
@ulferts apparently these plugin tests are never executed. shouldn't we move them?